### DISCUSSION TITLE
Theme's that keep the original button colors

### DISCUSSION THREAD

**Discussion by thibaultmol** [JulianPrieber/littlelink-custom#710] [Themes]
I like some of the other themes that are available but it's a shame that a lot of them just make the buttons monotone. 
Which is fine from a design perspective in some cases, but I personally really like the matching colors of the buttons.
While I'm sure you can modify the css file in the theme to fix this. I was wondering if maybe it'd be better if there was a naming convention for themes so that you for example have
Polygon
and
Polygon - Color

There's probably a better naming scheme for this. But having both a 'mono' verison and 'color' (button) version of some of the themes, would be nice. Just an idea

**Comment by JulianPrieber** [comment]
There is already an option for that in the theme config.  If you add `'use_default_buttons' => 'true',` to a theme config, the theme will attempt to use the default button layout.

![chrome_mQuWzOzlf1](https://github.com/LinkStackOrg/LinkStack/assets/60265788/1d1c8547-3904-47e7-848a-d288ae4e0dc1)

However, not all themes are compatible with this.

**Reply by thibaultmol** [reply]
ah, perfect! Guess something to add to the docs (did notice the docs don't actually say much about actually using the app. More about installing and maintaining the actual server aspect of it though)

**Reply by thibaultmol** [reply]
interesting.. the same bug happens as in your screenshot. Where the buttons go the left instead of being centered when you add that line to the config. Can replicate on other themes as well
